### DISCUSSION TITLE
how can I access the app created through "python manage.py create_app "new app" ".

### DISCUSSION THREAD

**Discussion by nte-578** [saleor/saleor#10739] [General]
Good afternoon all,
I have started using saleor recently and My requirement is to build an external app which will communicate the saleor core,
I created the app as per the documentation.
**_python manage.py create_app "Order App" \
   --permission MANAGE_USERS \
   --permission MANAGE_ORDERS \
   --target_url https://your-order-app/your-register-endpoint/_**

it did create an app and I got the **Auth code** but I am unable to find the app in the saloer folder.
can anyone tell me how can I access the app.

**Comment by IKarbowiak** [comment]
Hi @nte-578,

This command just creates the `App` instance in our database that pointed to your external `app` and allows you to communicate with it. You can find all information about apps [here](https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts) in our docs. I hope it will help you to understand it properly.
